router/diag_cntl.h

```diff
@@ -35,6 +35,21 @@
 #include "peripheral.h"
 #include "masks.h"
 
+#define DIAG_CNTL_CMD_DIAG_ID			33
+#define MAX_DIAGID_STR_LEN			30
+#define MIN_DIAGID_STR_LEN			5
```

Comment:
Per spec, I dont think there is an actual MAX/MIN STR LEN. We can keep it for now I guess, but may be something we have tor revisit. I am a little worried about making assumptions on this in the code

Reply:
okay, I'll record this. Maybe it would be helpful to figure out the overall packet size limit。